addons/sale/models/sale_order.py

```diff
@@ -2185,7 +2177,13 @@ def _update_order_line_info(self, product_id, quantity, **kwargs):
                 'product_uom_qty': quantity,
                 'sequence': ((self.order_line and self.order_line[-1].sequence + 1) or 10),  # put it at the end of the order
             })
-        return sol.price_unit
```

Comment:
We did it like that to speed up the request and avoid recomputing the price once again.
Wouldn't `sol.price_unit * (1 - (sol.discount or 0.0) / 100.0)` work instead, to keep the perf gain ?